tools/codemod/src/v4.0.0/__testfixtures__/full.input.tsx

```diff
@@ -1,7 +1,7 @@
 /*---------------------------------------------------------------------------------------------
-* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
-* See LICENSE.md in the project root for license terms and full copyright notice.
-*--------------------------------------------------------------------------------------------*/
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
```

Comment:
In addition to installing the extension you can fix current formatting inconsistencies for all files by running:
`rush prettier:fix`
This should get rid of changes like these :)

Reply:
You are correct, files under `__testfixtures__` are supposed to be ignored by prettier (see `.prettierignore`), not sure why this got formatted (for now you can revert this by hand).

Reply:
Git is not allowing me to commit the changes of removing the spaces. I keep getting this error: 
```
⚠ lint-staged prevented an empty git commit.
  Use the --allow-empty option to continue, or check your task configuration
```

Reply:
You are correct, `lint-staged` runs a git hook to update the copyright headers of changed files.
For now I've used `-n` https://git-scm.com/docs/git-commit#Documentation/git-commit.txt--n to get rid of this change, but we'll want to update the headers of `__testfixtures__` files in a separate PR.